src/sink/rest.rs

```diff
@@ -370,6 +370,7 @@ impl Sink for Rest {
                                     .await?
                             }
                             Err(e) => {
+                                error!("Error: {}", e);
```

Comment:
I don't think mentioning the artefact is necessary here, since this is the current error message: `[2021-03-10T07:17:36Z ERROR tremor_runtime::sink::rest] Error: CouldntResolveHost: couldn't resolve host name`.
Adding the action which resulted in the error sounds good!